packages/@okta/i18n/src/properties/login_nl_NL.properties

```diff
@@ -544,7 +546,7 @@ password.complexity.minAgeDays = Er moeten minimaal {0} dagen zijn verstreken si
 password.reset.verification = Verifieer met een van de volgende factoren om uw wachtwoord opnieuw in te stellen.
 
 # The actual description consists of the password length message + one or more list elements
-password.complexity.length = {0} tekens
+password.complexity.length = ten minste {0} tekens
```

Comment:
@mauriciocastillosilva-okta This is the Dutch mistranslation fix for [https://oktainc.atlassian.net/browse/OKTA-362310](url).